vignettes/package_structure.Rmd

```diff
@@ -22,124 +22,180 @@ To make it easier for new developers to join, this documentation is created.
 
 # Package Structures
 
-`mmrm` is an R package, and the following sections describe all files/folders that are contained in this package.
+`mmrm` is an R package, and the following sections describe all files/folders that are contained in this package as well as the repository.
 
-## data
+## `cache`
 
-The `data/` folder is where `mmrm` internal data is stored.
-See the [`fev_data` documentation](https://openpharma.github.io/mmrm/main/reference/fev_data.html) and the [introduction vignette](introduction.html#common-usage) for details.
+This folder contains cache files, and is not part of the package.
 
-## design
+## `data`
 
-The `design/` folder is where the prototypes and SAS implementations stored.
-When we have a new idea for a specific implementations, we can add our prototypes in a Rmarkdown file to this folder.
-We can then push these prototypes to github and collect feedback on the proposed ideas.
-Currently we only have SAS and TMB folders.
+The `data/` folder is where `mmrm` example data sets is stored.
+See the documentation for [`fev_data`](https://openpharma.github.io/mmrm/main/reference/fev_data.html) and [`bcva_data`](https://openpharma.github.io/mmrm/main/reference/bcva_data.html) for details.
 
-### SAS
+## `data-raw`
 
-The `design/SAS/` folder contains the programs to run MMRM models and corresponding SAS outputs.
-The results will be used for integration tests.
-See [integration tests](#how-to-add-a-new-integration-test?).
+The `data-raw/` folder contains source code to produce simulated example data sets. Currently this is only `bcva_data.R` (and its associated helper file) producing the `bcva_data` data set.
```

Comment:
hm I need to update this because I added the script to cache thee mmrm review comparisons